src/main/java/frc/robot/subsystems/AprilTag.java

```diff
@@ -72,36 +72,44 @@ public class AprilTag extends SubsystemBase implements ShuffleboardProducer {
       new EstimatedRobotPose(new Pose3d(), 0, List.of(), PoseStrategy.LOWEST_AMBIGUITY);
   private static final double LAST_RESULT_TIMEOUT = 0.1;
 
-  public static final Transform3d PRACTICE_ROBOT_TO_FRONT_CAMERA =
-      new Transform3d(new Translation3d(0.28, -0.26, 0.327), Rotation3d.kZero);
-  public static final Transform3d COMPETITION_ROBOT_TO_FRONT_CAMERA =
-      new Transform3d(new Translation3d(0.33, -0.26, 0.327), Rotation3d.kZero);
-
-  private static final Rotation3d REAR_CAMERA_ROTATION =
-      new Rotation3d(0, toRadians(-15), toRadians(180));
-  public static final Transform3d PRACTICE_ROBOT_TO_BACK_CAMERA =
-      new Transform3d(new Translation3d(-0.33, -0.255, +0.818), REAR_CAMERA_ROTATION);
-  public static final Transform3d COMPETITION_ROBOT_TO_BACK_CAMERA =
-      new Transform3d(new Translation3d(-0.36, -0.265, +0.80), REAR_CAMERA_ROTATION);
+  // TODO: verify ALL camera rotations and transforms.
+  private static final Rotation3d FRONT_RIGHT_CAMERA_ROTATION =
+      new Rotation3d(0, toRadians(20), toRadians(15));
+  public static final Transform3d PRACTICE_ROBOT_TO_FRONT_RIGHT_CAMERA =
+      new Transform3d(
+          new Translation3d(+0.274, +0.286, +0.197), FRONT_RIGHT_CAMERA_ROTATION);
```

Comment:
Shouldn't this be the same as the competition bot?
```suggestion
          new Translation3d(+0.274, -0.286, +0.197), FRONT_RIGHT_CAMERA_ROTATION);
```

Reply:
Yeah, that was a typo

Reply:
Good catch!